app/views/schools/participants/_currently_training_sorted_by_induction_start_date.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= render(Schools::Participants::Dashboard::SortControl.new(school:, selected: :induction_start_date)) %>
+
+<% participants.ects.each do |ect| %>
+  <%= govuk_summary_card(title: link_to_participant(ect.participant_profile, school)) do |card|
+    card.with_summary_list(
+        rows: [
+            { key: {
+                text: "Status",
+            },
+              value: {
+                  text: render(StatusTags::SchoolParticipantStatusTag.new(
+                      participant_profile: ect.participant_profile,
+                      induction_record: ect.induction_record,
+                      display_description: false,
+                      display_induction_start_date: true)),
+              },
+            },
+            { key: {
+                text: "Mentor",
+            },
+              value: {
+                  text: (
+                      if ect.mentored?
+                        link_to_participant(ect.mentor, school)
```

Comment:
@ltello can we change this to just show the mentor’s full name please, rather than being a link? Thanks!